### ISSUE TITLE
Icon Layout Module

### ISSUE THREAD

**Issue by thesublimeobject** [katiebushdesign/template#1]
Perhaps instead of adding icon layouts inside of the content module itself, make them a custom post type such that icons can be added as re-usable "components" that can then be inserted into the layout of the content modules, thus adding more flexible use-cases for repeatability in the same way the CPT Layouts are formed.

**Comment by bryanlilly** [comment]
Do you mean there would be a CPT called "Icon Layouts," then each layout type would be a post under that CPT?

Example:

**CPT** Icon Layouts 
 |_ **post** Boxes
 |_ |_ **field** icons

**Comment by thesublimeobject** [comment]
Honestly, I'm not totally sure yet. This was an idea I had about a month ago that made a ton of sense at the time, but I haven't worked much with the template since then, so I was more or less adding it so I don't forget to come back to the idea.

**Comment by bryanlilly** [comment]
Thinking about it a bit more, and I think what I said above would be cumbersome. If we used a "box" layout on another section with different icons, we'd have to duplicate the post and give it another name.

It could be
**CPT** Icons
|_ **post** icon
|_ **post** icon2

Then have a module in content that allows you to select the type and use a relationship to select the icons. It's not fundamentally different than what we currently do, but we wouldn't have to go in and update the icon list in the custom fields every time.

**Comment by thesublimeobject** [comment]
Right, _not_ updating the icon list would be one of the major advantages to doing something like this. At the time, there was an even bigger advantage, although again, I'll have to go in and try to remember what is was.
